src/lib/jsonParseV2/parseAttributeValue.cpp

```diff
@@ -49,7 +49,8 @@ std::string parseAttributeValue(ConnectionInfo* ciP, ContextAttribute* caP)
   if (document.HasParseError())
   {
     alarmMgr.badInput(clientIp, "JSON parse error");
-    oe.fill(SccBadRequest, "Errors found in incoming JSON buffer");
+    oe.reasonPhrase = ERROR_STRING_PARSERROR;
```

Comment:
Yeah, I know. We "fixed" the text.
However, you still have no oe.code. You might need one.

To "fix" your problem, we could add a third parameter to the constructor, and default it to NULL:

    OrionError::OrionError(HttpStatusCode _code, const std::string& _details, const char* reason = NULL)
    {
        code          = _code;
        reasonPhrase  = (reason == NULL) httpStatusCodeString(code) : reason;
        details       = _details;
    }